Looms.PoS/Looms.PoS.Application/Models/Responses/PaymentProvider/PaymentProviderResponse.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Looms.PoS.Domain.Enums;
+
+namespace Looms.PoS.Application.Models.Responses.PaymentProvider;
+
+public class PaymentProviderResponse
```

Comment:
```suggestion
public record PaymentProviderResponse
```

Reply:
Can you add it as a separate task, because most of our responses are defined using class?